nimcr.nim

```diff
@@ -64,7 +64,12 @@ if not exeName.existsFile or filename.fileNewer exeName:
 
 # Run the target, or show an error
 if buildStatus == 0:
-  quit execShellCmd(exeName & " " & args[filenamePos + 1 .. ^1].join(" "))
+  let p = startProcess(exeName,  args=args[filenamePos+1 .. ^1], 
+                       options={poStdErrToStdOut, poParentStreams, poUsePath})
+  while p.running():
+    sleep(1)
```

Comment:
This could potentially add a second long delay for very short scripts.

Reply:
That would be a second when you use `sleep()` in C or `time.sleep()` in Python. In Nim `os.sleep()` takes miliseconds as a parameter (https://nim-lang.org/docs/os.html#sleep%2Cint), which I thought would be acceptable (best would be to block until the command exits, but I have not found a way to do that).

Reply:
Ah right, it won't sleep for more than a millisecond. Maybe [`waitForExit`](https://nim-lang.org/docs/osproc.html#waitForExit%2CProcess%2Cint) is what you are looking for though?

Reply:
That would work, especially since I already use `poParentStreams`. Now to figure out how to amend a PR...